src/components/AppDrawer/AppDrawer.js

```diff
@@ -124,6 +124,10 @@ function AppDrawer() {
     }
   };
 
+  const handleSave = () => {
+    onClose();
+    setForceRerender(!forceRerender);
```

Comment:
this should be always false

Reply:
if we want to rerender the page. that value should be update, if we update same value then page will not gonna rerender.

Reply:
We are using this prop to update the state only - there is no other use of this prop?